tests/native_tests/native_macros.hy

```diff
@@ -403,6 +401,7 @@ in expansions."
   (assert (= "This is the local version of `nonlocal-test-macro` returning 3!"
              #test-module-tag-2 3)))
 
+#@(pytest.mark.xfail
```

Comment:
Why did this test start failing?

Reply:
See the commit message.

Reply:
Hmm, I think I disagree with the (xfailed) assertion on line 447, but I'll make a separate issue for that.